components/inputs/input-styles.js

```diff
@@ -0,0 +1,72 @@
+import '../colors/colors.js';
+import { css } from 'lit-element/lit-element.js';
```

Comment:
The obvious major changes in here [compared to the Polymer version](https://github.com/BrightspaceUI/inputs/blob/master/d2l-input-shared-styles.js):

- Most of the CSS custom properties are gone, except for those that were actively being used. That means that I won't be removing them from the old code since some places are referencing them, but I didn't see a point to keeping them around. They were just letting people customize things that we probably don't want them customizing.
- All the targeted rules (e.g. `input[type="email].d2l-input`) are gone. Those were needed for the shady CSS polyfills -- probably the older v0 version -- and after testing in Edge and IE11 I've deemed them no longer required.
- Obviously I've collapsed all the style down and removed the style mixins